application/src/main/java/org/togetherjava/tjbot/commands/tag/TagManageCommand.java

```diff
@@ -127,7 +127,7 @@ private void editIdTag(SlashCommandEvent event) {
                     && ex.getErrorResponse() == ErrorResponse.UNKNOWN_MESSAGE) {
                 event.reply("This message doesn't exist").setEphemeral(true).queue();
             } else {
-                throw new RuntimeException(failure);
+                throw new RethrownRuntimeException(failure);
```

Comment:
Instead of adding a useless custom exception you can also just tell Sonarcloud to ignore the warning

Reply:
How?

Reply:
https://stackoverflow.com/questions/39109228/how-can-we-ignore-some-sonarqube-rules-in-java/39109612#39109612